src/commands/dev/gcs/headers.rs

```diff
@@ -7,15 +7,19 @@ use hyper::http::request::Parts as RequestParts;
 use hyper::http::response::Parts as ResponseParts;
```

Comment:
this page could  maybe  benefit from a top level comment explaining why headers need to be  handled a special way. that may be clearer than comments  on individual fns